chainer/functions/connection/convolution_nd.py

```diff
@@ -421,3 +421,49 @@ def convolution_nd(x, W, b=None, stride=1, pad=0, cover_all=False):
         return func(x, W)
     else:
         return func(x, W, b)
+
+
+def convolution_1d(x, W, b=None, stride=1, pad=0, cover_all=False):
+    """1-dimensional convolution function.
+
+    .. note::
+
+        This function calls :func:`~chainer.functions.convolution_nd`
+        internally, so see the detail in the documentation of
+        the documentation of :func:`~chainer.functions.convolution_nd`.
```

Comment:
the repetiton of "the documentation" is typo?

Reply:
Oops, will fix it